azurerm/data_source_dns_zone.go

```diff
@@ -43,6 +43,29 @@ func dataSourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
+				Type:     schema.TypeString,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+			},
+
+			"registration_virtual_networks": {
+				Type:     schema.TypeList,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"resolution_virtual_networks": {
```

Comment:
since this is a list of ID's - I think this'd be better as `resolution_virtual_network_ids`

Reply:
good idea.

Reply:
done.